### PR TITLE
ldc 1.0.0-alpha1 (devel)

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Installation is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/42456/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/install___devel_ldc/

Please keep the PR template in PRs, thanks.

File: Library/Formula/ldc.rb [hunk 1]
```diff
@@ -12,19 +12,39 @@ class Ldc < Formula
...
+    url "https://github.com/ldc-developers/ldc/releases/download/v1.0.0-alpha1/ldc-1.0.0-alpha1-src.tar.gz"
+    sha256 "b656437d0d7568c5ac4ef4366376184c06013e79f3dd5a512b18ca9f20df4b63"
+    version "1.0.0-alpha1"
+  end
+
   needs :cxx11
 
   depends_on "cmake" => :build
   depends_on "llvm" => :build
+  depends_on "dmd" => :build
```
Comment (DomT4) [inline Library/Formula/ldc.rb]: If this is only needed for the `devel` build it should be tucked into the `devel` block above, one empty line underneath the `version` in this case.

**Comment by John-Colvin** [comment]
pushed a new, better version. In the future there won't be a special case for stable, it'll just have its own `"ldc-lts"` resource.

**Comment by tdsmith** [comment]
Thanks for your pull request! :tada: Homebrew thrives because of contributions like this one and we appreciate it.

**Pull request closed without merging**